### ISSUE TITLE
Error /vendor/laravel/framework/src/Illuminate/Console/Scheduling/ScheduleListCommand.php

### ISSUE THREAD

**Issue by Sebastiangperez** [laravel/framework#40317]
- Laravel Version: 8.78.1

- PHP Version: 8.0.8
- Database Driver & Version: 10.5.13-MariaDB

### Description:
Trying to run php artisan schedule:list and get an error that TimeZone is not correct. 
I have to change this : 
`                (new CronExpression($event->expression))
                            ->getNextRunDate(Carbon::now()->setTimezone($event->timezone))
                            ->setTimezone($this->option('timezone', config('app.timezone')))
                            ->format('Y-m-d H:i:s P'),`
To this : 
```
                (new CronExpression($event->expression))
                            ->getNextRunDate(Carbon::now()->setTimezone($event->timezone))
                            ->setTimezone(new DateTimeZone(config('app.timezone')))
                            ->format('Y-m-d H:i:s P'),
```
And works : 

### Steps To Reproduce:
Edit config/app.php , and change time zone to America/Argentina/Buenos_Aires

**Comment by seriquynh** [comment]
This is fixed in https://github.com/laravel/framework/pull/40304

Just wait for the next release!

**Comment by Sebastiangperez** [comment]
> This is fixed in #40304
> 
> Just wait for the next release!

Sorry , is that i had a challenge for a job interview and when i had this issue , i didn't  know what happend.
Thanks !
